docs/contributing/testing_guide.rst

```diff
@@ -9,6 +9,50 @@ Testing Guide
    :local:
    :backlinks: none
 
+Summary
+=======
+
+* New functionality added to PlasmaPy must also have tests.
+
+* Tests are located in files that begin with :file:`test_` which are
+  inside subdirectories named :file:`tests`.
+
+* Tests are either functions beginning with ``test_`` or classes
+  beginning with ``Test``.
+
+* To install the packages needed to run the tests:
+
+  - Open a terminal.
+
+  - Navigate to the top-level directory (probably named
+    :file:`PlasmaPy`) in your local clone of PlasmaPy's repository.
+
+  - If you are on MacOS or Linux, run:
+
+    .. code-block:: console
+
+       python -m pip install -e.[tests]
+
+    If you are on Windows, run:
+
+    .. code-block:: console
+
+       py -m pip install -e .[tests]
+
+    These commands will perform an `editable installation`_ of your
+    local clone of PlasmaPy.
+
+* Run ``pytest`` in the command line in order to run tests in that
+  directory and its subdirectories.
+
+* Here is an example of a minimal software test that uses an
```

Comment:
```suggestion
* Here is an example of a minimal ``pytest`` test that uses an
```
because this being a software test is inferrable from context, but this wouldn't be compatible with nosetest.